carma_wm/src/SignalizedIntersectionManager.cpp

```diff
@@ -381,4 +384,12 @@ namespace carma_wm
 
     return *this;
   }
+
+  SignalizedIntersectionManager::SignalizedIntersectionManager(const SignalizedIntersectionManager& other)
```

Comment:
Out of curiosity was there a place where the copy constructor was being used?

Reply:
Yes, the compiler complained that the implicit copy constructor may have been doing something undesirable.  Let me check what that was.